superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { Button, Nav, Navbar, MenuItem } from 'react-bootstrap';
```

Comment:
We may also want to decide whether to keep creating new UI components based on Bootstrap. To me, this seems like a great opportunity to migrate away from it.

What do you think, @rusackas @kristw ?

Reply:
To me migrating away from bootstrap is an important decision with intricate considerations that are outside the scope of this PR. We should start a `[DISCUSS]` thread on the mailing list on this topic.

Reply:
There were some discussion when [SIP-34](https://github.com/apache/incubator-superset/issues/8976#issuecomment-590565931) was closed and I thought the general consensus is we want to at least move away from Bootstrap 3?

If we are to implement new UI components to new UI, it'd be the best to start clean and reduce dependencies on Bootstrap as much as possible, especially if the component doesn't need to reuse much styling from Bootstrap anyway.